docs/README.md

```diff
@@ -77,12 +84,13 @@
   ```
 
 ## Step 3 - Use MyProvider.psm1
-- Import-Module SHiPS (See [`Installing SHiPS` ][readme] for details)
+
+- Import-Module SHiPS (See [`Installing SHiPS`][readme] for details)
```

Comment:
Should "Installing SHiPS" be inside `` ?

Reply:
fixed. Will update the PR.